nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -377,42 +440,79 @@ define([
          * 1. if there's a tab showing, stop() it and detach it
          * 2. update the tabs state to be selected
          * @param {string} tab id of the tab to display
+         * @param {string} fileType id of the filetype we're swapping to
          */
-        function toggleTab(tab) {
+        function toggleTab(tab, fileType) {
```

Comment:
not sure what happened here, but on initial load of the cell i'm now getting no tab selected (rather than default of configure).